pages/index.tsx

```diff
@@ -10,6 +11,7 @@ export default function Home() {
         <meta name="description" content="Generated by Taodemy" />
       </Head>
       <main className="page__wrapper">
+        <Header />
```

Comment:
Do you think the Header component is better to placed here? Please consider we have several other pages that have the same header, at least for now.

Reply:
I think the global layout is needed. plz refer to:
![image](https://user-images.githubusercontent.com/73293255/219283805-4a71fa10-8b77-44ec-bbeb-42de4fbb4fd8.png)

Reply:
Thank you Cam I have added the  global layouts